server/controllers/user.js

```diff
@@ -33,6 +34,14 @@ export const addUser= async(req, res) => {
     dateOfBirth:dateOfBirth,
     flightNumbers:flightNumbers
   });
+  if(!validator.isEmail(Email)){
+    res.status(400).json('Error: Invalid Email');
+    return;
+  }
+
+if(!validator.isDate(dateOfBirth)){
```

Comment:
Validator not working with correctly formated date : 2021-11-14T20:23:01.000Z
also must return after every response is sent  to avoid : UnhandledPromiseRejectionWarning: Error [ERR_HTTP_HEADERS_SENT]: Cannot set headers after they are sent to the client

Reply:
I feel that validating the format here is redundant since you already force it in your front end code. this code works to check whether it is a valid date or not.

Reply:
I think validating the format here is redundant since you force it in the frontend already so i just validated whether it is a date or not and I believe it is sufficient and we can move to more important functionalities instead of getting stuck on things of secondary importance (which is already implemented).

Reply:
the return has been added.

Reply:
okay remove the date validation so I can merge the pr, as  now it's preventing any date from being added.

Reply:
done